examples/common/entry/entry_sdl.cpp

```diff
@@ -726,23 +708,6 @@ namespace entry
 						}
 						break;
 
-					case SDL_JOYBUTTONDOWN:
-					case SDL_JOYBUTTONUP:
-						{
-							const SDL_JoyButtonEvent& bev = event.jbutton;
-							GamepadHandle handle = findGamepad(bev.which);
-
-							if (isValid(handle) )
-							{
-								Key::Enum key = translateGamepad(bev.button);
```

Comment:
JoyButtons are different than ControllerButtons. translateGamepad produces wrong key for some joy buttons.
This affects ThumbLeft/Right, ShoulderLeft/Right, Back, Start, and Guide.